torch/_export/serde/serialize.py

```diff
@@ -134,20 +193,65 @@ def serialize_metadata(node: torch.fx.Node) -> Dict[str, str]:
     return ret
 
 
+def deserialize_metadata(metadata) -> Dict[str, str]:
+    ret = {}
+    if stack_trace := metadata.get("stack_trace"):
+        ret["stack_trace"] = stack_trace
+    # Need an explicit None check instead of walrus operator, because
+    # module_fqn can be the empty string if the node belongs to the root.
+    # The walrus operator returns False on an empty string :(
+    module_fqn = metadata.get("module_fqn")
+    if module_fqn is not None:
+        ret["module_fqn"] = module_fqn
+    # TODO(angelayi) add nn_module_stack and source_fn
+    return ret
+
+
 def serialize_operator(target) -> str:
     if isinstance(target, str):
         return target
-    elif target in _SYM_INT_OPS:
-        return f"{target.__module__}.{target.__name__}"
-    elif isinstance(target, torch._ops.HigherOrderOperator):
-        return target.__name__
+    return f"{target.__module__}.{target.__name__}"
+
+
+def deserialize_operator(serialized_target: str):
+    module = None
+    if serialized_target.startswith("_operator"):
```

Comment:
Can we move this kind of parse / print contract into a class like `OperatorSerde`? Otherwise this may get out of sync pretty easily...

Reply:
Hmm, test failures are pointing to the parsing logic here, so additional evidence that this part of the code seems brittle.

Reply:
hm, i think it's because removeprefix does not exist in python 3.8